buildSrc/src/main/groovy/Docker.groovy

```diff
@@ -652,10 +652,10 @@ class Docker {
                     copySpec.into 'src'
                 }
             }
-            config.imageName = "${imgName}:local-build"
+            config.imageName = "${imgName}:${LOCAL_BUILD_TAG}"
```

Comment:
This is a funny inconsistency - buildPyWheel() invocations already have the `deephaven/` prefix?

EDIT: Nope, not any more they don't - this is a dead method I think? Can we remove it instead?

Reply:
Removed